testing/decommission-test.sh

```diff
@@ -59,12 +59,13 @@ function main() {
 
     echo -e "\n\n"
     echo "decommission-test.sh: main(): Verify data after decommission:"
-    execute_pod_script verify-data.sh "${DEBUG_POD_NAME}"
-    RESULT=$(read_script_result default "${DEBUG_POD_NAME}" verify-data.log)
+    RESULT="script failed"
+    while [ "$RESULT" != "script passed" ]; do
```

Comment:
isn't this actually an infinite loop to wait? also should we have a sleep between each run?

Reply:
Yes, Shubhendu, that is a great idea. We should avoid infinite loops and include pauses between iterations.